src/services/StellarContractService.ts

```
@@ -0,0 +1,291 @@
+      logger.error("Payment processing failed:", error);
+      if (error instanceof AppError) {
+        throw error;
+      }
+      throw new AppError("Failed to process payment", 500);
+    }
+  }
+
+  /**
+   * Get merchant details including supported tokens
+   */
+  async getMerchantDetails(merchantAddress: string): Promise<MerchantDetails> {
```

ESLint

[error] 'MerchantDetails' is not defined.

(no-undef)